data/tribes/initialization/empire/units.lua

```diff
@@ -1551,6 +1817,12 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_ferry_yard",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Ferry Yard
+            lore = pgettext("empire_building",
+               "‘I’m not Charon, this is not the Styx, Cerberus doesn’t guard the other side,"..
+               " but I still can’t take any living creature across the river.’"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Ferry Yard
+            lore_author = pgettext("empire_building", "Complaint of the lone ferryman"),
```

Comment:
I've found a Latin quote that may be OK:
"Non terretur fluminibus, qui maria transiit" / "Who crossed seas does not fear rivers"

What do you think?